test.sh

```diff
@@ -15,10 +15,16 @@ trap finish EXIT
 
 echo "> Installing helm chart, waiting until app is ready..."
 dataKey="$(docker run --rm cyberark/conjur data-key generate)"
+
+# NOTE: external load balancers and persistent volumes are not always available
+# so they are turned off.
 helm install --wait \
-             --timeout 900 \
+             --timeout 180 \
              --name $RELEASE_NAME \
-             --set "dataKey=$dataKey" ./conjur-oss
+             --set "dataKey=$dataKey" \
+             --set "service.external.enabled=false" \
+             --set "postgres.persistentVolume.create=false" \
```

Comment:
Since the change here reduces the confidence in our E2E coverage, can we have the default run test the old flags and the actions test the reduced set?

Reply:
@sgnn7 This script is being run only in this pipeline AFAICT. I'm thinking that the permutations of values that we are interested in could each be a separate test. For now this is the only test that will run because this is the only pipeline we have. The idea being that once a pipeline on GKE is brought in then persistent volume and external services can have their own tests

Reply:
This is the only _automated_ pipeline that this code runs in and no-storage PG is not the most common deployment scenario. I would still like to see the manual test run to default to PV-backed DB and this pipeline can run it without.